backend/src/libs/packages/open-ai/open-ai.package.ts

```diff
@@ -80,15 +83,20 @@ class OpenAi extends BaseHttpApi {
       ),
       {
         method: 'POST',
-        payload: JSON.stringify({ prompt, number, size }),
+        payload: JSON.stringify({
+          prompt,
+          n: number,
+          size,
+          response_format: this.defaultImageGenerateConfig.response_format,
+        }),
         token: this.apiKey,
         contentType: ContentType.JSON,
       },
     );
 
     const [response] = data.data;
 
-    return response?.url ?? null;
+    return response?.b64_json ?? null;
```

Comment:
```suggestion
    return response?[this.defaultImageGenerateConfig.response_format]?? null;
```

Reply:
we don't need this, it's defined in type